ltr/client/opensearch_client.py

```diff
@@ -0,0 +1,263 @@
+import os
+import requests
+
+from .base_client import BaseClient
+from ltr.helpers.handle_resp import resp_msg
+
+import json
+from opensearchpy import OpenSearch
+from opensearchpy import helpers
+
+class ElasticResp():
```

Comment:
Might this be better named OpenSearchResp?

Reply:
yes! will do.

Reply:
updated